src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrdersSearch.js

```diff
@@ -25,34 +46,71 @@ function PurchaseOrdersSearch() {
             'orderNumber'
         )
     );
-    const searchLoading = useSelector(state =>
-        collectionSelectorHelpers.getSearchLoading(state.purchaseOrders)
-    );
+    const [orderNumber, setOrderNumber] = useState('');
+    const search = () => history.push(`/purchasing/purchase-orders/${orderNumber}/`);
```

Comment:
if the search function just takes you straight to an order, then how do you ever get a searchResults list to render on this page (like you've started rendering on line 87

Reply:
I'm using the collectionSelectorHelper's getSearchItems method on line 40 based on the order number and it's limited to 100 results. Maybe the naming could be a little better.